src/api/axios.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
+
+const isBundled = process.env.NODE_ENV === 'production'
+const prod = 'https://wps-api-dev-6-secure-auzhsi.pathfinder.gov.bc.ca'
```

Comment:
Hard coding the production URL seems very nasty, can we not get something a but more dynamic in here?

Reply:
Yes eventually we need to pull urls from the environment variable but it was kept like that for the time being